CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   - `core.watchman.register_snapshot_trigger`
   - `diff.format`
 
+* `jj` now ignores `$PAGER` set in the environment and uses `less -FRX` on most
+  platforms (`:builtin` on Windows). See [the docs](docs/config.md#pager) for
+  more information, and [#3502](https://github.com/jj-vcs/jj/issues/3502) for
+  motivation.
```

Comment:
nit: since new version was released, you'll need to rebase this and move the changelog entry before merging.